src/pug/blog/shop_opening.pug

```diff
@@ -0,0 +1,54 @@
+extends _blog.pug
+
+block append subtitle
+  | Shop opening
+
+block content
+  h2 Shop opening
+  +news_header('22 February 2023', 'report_8', null)
+
+  h3 Input Labs Distribution
+  p It may not be a grand opening, and it is not really a traditional shop either. We are launching #[+ax('https://inputlabs.squarespace.com') Input Labs Distribution], a portal that we will use to organize better the distribution of:
+  ul
+    li Free kits for testers, as part of our #[+a('hardware_testing') Testers program].
+    li #[+a('hardware_review') Review units] for the media, such as Youtube content creators and websites.
+    li Selected DIY kits, components and materials to everybody. #[em This is the new thing!]
+  hr
+
+  h3 Why are we opening a shop?
+  p First and foremost because we realized that there is a massive amount of people interested in the project without access to 3D-printers, soldering equipment, etc... which is a bit of a #[em duh] moment, but we did not expect so much traction from non-DIY-users at first. The pressure was so high that we created a dedicated room on Discord (#[em diy-share-parts]) so the community members could help each other by sharing spare parts, making group orders, and offering their time and expertize so everyone has access to a controller if they want. But this solution is only a workaround, we cannot rely on community members working for free to make our accessible controllers reach everyone.
+
+  p Logistics and distribution is not an easy task, we have over #[em 30 testers] now, for some of them we create parts for free but they take care of the shipping costs, organizing this through a shop interface with #[em 0 euros] articles simplifies the process a lot.
+
+  p Additionally it is not environmentally friendly (nor cheap) for everyone to place minimum orders for components, for example the minimum amount of PCBs to order in JLCPCB is 5 (2 of them soldered and 3 of them raw), and the minimum amount of Protopasta conductive filament to buy is 50 grams while the Alpakka reference design only needs 4 grams, so quite a waste in general. On the top of that, some of the components are almost impossible to source in some regions, for example getting M2.5 #[em millimeters] bolts in the United States.
```

Comment:
Rephrase: Divide into two sentences: "... for components. For example ..."
"(two with soldered SMD components, three without)"
"and" => "or"
"millimeters" => "metric" (why emphasis?)